### PR TITLE
Completed Javascript Bank Challenge - Daniel Rosenbaum

### PR DESCRIPTION
**Pull request by danielwrosenbaum**
Screen Shot 2021-02-16 at 8 53 57 PM

### CODE REVIEW

File: javascript-bank/account.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Account.prototype.getBalance = function () {
+  var total = 0;
+  for (var i = 0; i < this.transactions.length; i++) {
+    if (this.transactions[i].type === 'deposit') {
+      total += this.transactions[i].amount;
+    } else if (this.transactions[i].type === 'withdraw') {
+      total -= this.transactions[i].amount;
+    } else {
+      return 0;
+    }
```
Comment (codingcodymiller) [inline javascript-bank/account.js:38]: This can be removed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Good work so far!! One minor note for you:

File: javascript-bank/account.js (hunk 1 above)
Comment (danielwrosenbaum) [reply javascript-bank/account.js:38]: oh duh! totally missed the fact that my total was already set to 0 and would return that if conditionals weren't met!

**Review by codingcodymiller — APPROVED** [review]
🎉 🎉 🎉

**Pull request merged**